examples/hpc/dla-driver-ffi/src/lib.rs

```diff
@@ -9,28 +9,49 @@ use alloc::vec::Vec;
 use core::ffi::{c_char, CStr};
 use core::slice;
 use dla_driver::layers::{conv2d, conv2d_bias, conv2d_bias_relu, conv2d_relu};
-use dla_driver::tensor3::{Order3, Tensor3};
+use dla_driver::tensor3::{rescale, Order3, Tensor3};
 use dla_driver::tensor4::{Order4, Tensor4};
 use dla_driver::{Padding, Stride};
 
 /// Converts C-types to DLA Tensors for use with the highlevel layer
+///
+/// # Arguments
+///
+/// * `input_data` - Input data  
+/// * `input_channels` - Number of channels in the input
+/// * `input_height` - Height of the input
+/// * `input_width` - Width of the input
+/// * `input_order` - Layout of the input data
+/// * `input_zero` - Possible zero point correction value
+/// * `kernel_data` - Kernel/weight data
+/// * `kernel_amount` - Number of output channels
+/// * `kernel_channels` - Number of input channels
+/// * `kernel_height` - Kernel height
+/// * `kernel_width` - Kernel width
+/// * `kernel_order` - Kernel data layout
```

Comment:
That's a little excessive :sweat_smile:  I meant to document the ones that are unobvious from context. I think documenting that `kernel_height` means "Kernel height" is just unnecessary inertia.

Reply:
Same goes for other repetitions of this pattern.

Reply:
Removed unnecessary documentation